hathor/nanocontracts/blueprint_env.py

```diff
@@ -164,26 +164,26 @@ def can_melt(
     @final
     def call_public_method(
         self,
-        nc_id: ContractId,
-        method_name: str,
-        actions: list[NCAction],
+        _nc_id: ContractId,
```

Comment:
Should we forbid arguments starting with `_` in Blueprint methods?

Reply:
I think that's fair.